### ISSUE TITLE
search numbers

### ISSUE THREAD

**Issue by rschiavi** [aslagle/reactive-table#226]
should searching numbers work?

**Comment by aslagle** [comment]
Yes, if the collection contains numbers. If you're storing them as strings, it'll match if the search term is anywhere in the string.
